kpi/management/commands/provision_server.py

```diff
@@ -103,7 +103,7 @@ def handle_socialapp(self, kwargs):
         provider_id = kwargs['provider_id']
         name = kwargs['name']
         client_id = kwargs['client_id']
-        secret = kwargs.get('secret', os.getenv('SOCIAL_APP_SECRET', ''))
+        secret = kwargs.get('secret', '') or os.getenv('SOCIAL_APP_SECRET')
```

Comment:
I know I'm harping on this but it could be more clear. 

The second arg in the get function is the default. Which defaults to `''` which it then considered "falsey" and so the os.getenv('SOCIAL_APP_SECRET') gets evaluated.

It can be simplified as ` kwargs.get('secret', os.getenv('SOCIAL_APP_SECRET'))`.